test/utils/asserts.hpp

```diff
@@ -23,45 +23,56 @@ MATCHER(Result, "empty result")
     return !arg.trapped && !arg.has_value;
 }
 
-template <typename T, typename... Ts>
-constexpr bool is_any_of = std::disjunction_v<std::is_same<T, Ts>...>;
-
 MATCHER_P(Result, value, "")  // NOLINT(readability-redundant-string-init)
 {
     using namespace fizzy;
 
-    static_assert(is_any_of<value_type, uint32_t, int32_t, uint64_t, int64_t, float, double>);
```

Comment:
Why remove this?

Reply:
Not strictly needed now and the Result() will just return false.